### PR TITLE
Run tox on GitHub Actions

### PR DESCRIPTION
**Pull request by hroncok**

### CODE REVIEW

**Comment by hroncok** [comment]
Results can be observed at https://github.com/hroncok/zombie-imp/actions

**Comment by hroncok** [comment]
Apparently, our container does not have python3.X-test in it :/

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  tox_test:
+    name: Tox test
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run Tox tests
+      id: test
+      uses: fedora-python/tox-github-action@main
+      with:
+        tox_env: ${{ matrix.tox_env }}
+        dnf_install: python3.10-test python3.11-test python3.12-test
```
Comment (hroncok) [inline .github/workflows/main.yml:17]: I wonder if this could be part of the matrix to avoid delays on other Pythons.

**Comment by encukou** [comment]
> Huhuhuhu, yessss!

It grows stronger...

**Pull request merged**